src/Catalog/Monitoring/PackageTimestampMetadata.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
```

Comment:
definitely...no idea how it got in here, I think VS accidentally cloned the file.